### ISSUE TITLE
Performance of pretrained model is not good 

### ISSUE THREAD

**Issue by Ian-Ren** [alexklwong/calibrated-backprojection-network#13]
Hi, Alex,  Thank you so much for the excellent work. I am using the pretrained model to run on VOID test dataset. However, the performance is pretty bad for all three cases. Here is a screen shot of the results.  I am wondering where the problem could be. (1) Does the pytorch version matters . Mine is cuda1.8.1 + cuda11.1.  (2) Is the pretrained model on share drive the final version used in the paper ?   Thank you very much. 

Ian
![Capture](https://user-images.githubusercontent.com/52585522/159838296-7719cdfb-abd3-4a6c-80a8-34f0450ed659.PNG)

**Comment by Ian-Ren** [comment]
Never mind. It is the problem of my python configuration issue. Sorry for that

**Comment by alexklwong** [comment]
Can you detail the steps that caused the accuracy to be 6000+ MAE? This is just in case others run into a similar problem.

**Comment by Ian-Ren** [comment]
I think this problem is quite specific for me. I am using lab's server. Its has two pytorch installed. One through Anaconda, one through pip. Seems the one through pip is not correctly installed.  However, the python linked with Anaconda is not added to system path.  When I run the program, it invoke the natives Python installed on Ubuntu instead of the one installed by Anaconda.  Thus even though the program gives no error, but the results is not correct.  I correct this by adding anaconda python into system path. The results is good.  Hope it helps
